CONTRIBUTING.md

```diff
@@ -142,99 +142,77 @@ This will serve the app at [localhost:4141](http://localhost:4141/), and watch f
 
-```bash
-cd /path/to/kedro-viz
-```
+### Launch a development server with a real Kedro project
+
+> **Note**: Kedro-Viz>=3.8.0 will not work with projects created with Kedro<=0.16.6. Please consider migrating your project to Kedro>=0.17.0 before you develop against the latest version of Kedro-Viz. 
 
-Install kedro-viz's development dependencies with:
+Before launching a development server with a real Kedro project, you'd need to have [Python](https://www.python.org/)(>=3.7, <3.9) installed. We strongly recommend setting up [conda](https://docs.conda.io/en/latest/) to manage your Python versions and virtual environments. You can visit Kedro's [guide to installing conda](https://kedro.readthedocs.io/en/latest/02_get_started/01_prerequisites.html#conda) for more information.
 
+The Kedro-Viz repository comes with an example project in the `demo-project` folder. This is used on the [public demo](https://demo.kedro.org/). To use it in your development environment, you need to install both the Kedro-Viz dependencies and a minimal set of dependencies  for the demo project:
 ```bash
 pip3 install -r package/test_requirements.txt
+pip3 install -r demo-project/src/docker_requirements.txt
 ```
 
-Build the application with:
+Now build the application with:
 
 ```bash
 make build
 ```
 
-As far as the development server for the backend is concerned, you only need to run `make build` once when you first setup the project.
-
-Then launch the server with:
+As far as the development server for the backend is concerned, you only need to run `make build` once when you first setup the project. You can then launch the server with:
 
 ```bash
-make run PROJECT_PATH=<path-to-your-test-project>/new-kedro-project
+make run
 ```
-This command will launch a Kedro-Viz server at [localhost:4142](http://localhost:4142) and serve data from a real Kedro pipeline located at the project path supplied to the command. From then on, launching the app locally at [localhost:4141](http://localhost:4141) will pull data from the Kedro-Viz server that is running on port 4142. To avoid needing to supply `PROJECT_PATH` every time you call `make run`, you can set it as an environment variable:
+
+This command will launch a Kedro-Viz server at [localhost:4142](http://localhost:4142) and serve data from `demo-project`. From then on, launching the app locally at [localhost:4141](http://localhost:4141) will pull data from the Kedro-Viz server that is running on port 4142.
```

Comment:
I'm not sure how clear it is at this stage of the instructions that the backend and frontend are now running in tandem and that one is feeding the other.

Perhaps adding in another reference here to running the frontend with `npm start` will hammer home the point that now the backend and frontend are running together?

Do we need to be as explicit as saying something like, "You need to have two terminal windows open..." etc etc?

```suggestion
This command will launch a Kedro-Viz server at [localhost:4142](http://localhost:4142) and serve data from `demo-project`. From then on, launching the app locally with `npm start` at [localhost:4141](http://localhost:4141) will pull data from the Kedro-Viz server that is running on port 4142.
```

Reply:
This is a great point, thank you! I've made it clearer in 99c173a. Note that if you're just doing backend development, you can actually run the backend server at 4242 without the frontend one

Reply:
Great! Thanks for adding.